packages/common/test/signHelpers.test.ts

```diff
@@ -24,10 +28,12 @@ test('test verifyMessageSignature', async () => {
         message,
         {
             0: {
-                0: '5420cc5a6c956e8a9b47a834d8aaee1176349435f8d172f0255db28a96040eed553d18cd98e2c873bf37a0c1794d382397f68a230485efab1ef16f678008080a',
+                0: '445197d79ca90d8cc8440328dac9f307932ade0c03cc7aa575b59b746e26e5f1bca13ade5ff7a56e918ba5a32450fdf52b034cd2580929b21213263e81f7f809',
             },
         },
         {
+            accountAddress:
+                '3eP94feEdmhYiPC1333F9VoV31KGMswonuHk5tqmZrzf761zK5',
```

Comment:
It would be nice with a test where the `accountAddress` here is incorrect, to verify that it fails to verify the signature in that case.

Reply:
Yes. Added one with the wrong accountAddress, and one where the signature is incorrect.